airflow/gcp/hooks/dlp.py

```diff
@@ -531,9 +552,13 @@ def delete_deidentify_template(
         project_id = project_id or self.project_id
 
         if organization_id:
-            name = DlpServiceClient.organization_deidentify_template_path(organization_id, template_id)
+            name = DlpServiceClient.organization_deidentify_template_path(
+                organization_id, template_id
+            )
```

Comment:
Are these kind of changes required here (shortening the lines)? Since line pylint `max-line-length=110` it should be fine as it was. If you use **Black** I think you should set line length to 110 as well.

Reply:
I think I set it to 79 as PEP-8 suggested. I'll change it to 110 for this project.

Reply:
**Black** by default is set to 88. This value is not obvious. Here is explanation: https://github.com/psf/black#line-length